packages/web-pkg/src/components/TextEditor/TextEditor.vue

```diff
@@ -110,11 +114,14 @@ export default defineComponent({
       }
     })
 
+    const sanitize = (html) => dompurify.sanitize(html)
```

Comment:
this doesn't seem to sanitize on initial rendering when I enter the malicious html... e.g `<a href="http://example.com/page.html?userInput=<script>alert('Injected!');</script>">test</a>`
only on subsequent renderings after page reload

Reply:
Bad, then let's sanitize the currentContent(prop) as well.

Reply:
Can you pls check again ?